### PR TITLE
Update middleware-database-error-page-obsolete.md

### PR DESCRIPTION
**Pull request by Rick-Anderson**
## Summary

Describe your changes here.

Fixes #Issue_Number (if available)

### CODE REVIEW

File: includes/core-changes/aspnetcore/5.0/middleware-database-error-page-obsolete.md [hunk 1]
```diff
@@ -56,6 +56,8 @@ Complete the following steps:
         services.AddDatabaseDeveloperPageExceptionFilter();
     }
     ```
+    
+1. Add the `Microsoft.AspNetCore.Diagnostics.EntityFrameworkCore` NuGet package to the project file.
```
Comment (IEvangelist) [inline includes/core-changes/aspnetcore/5.0/middleware-database-error-page-obsolete.md:60]:
```suggestion

1. Add the `Microsoft.AspNetCore.Diagnostics.EntityFrameworkCore` NuGet package to the project file.
```

Remove whitespace that is causing MD lint failure

**Review by IEvangelist — CHANGES_REQUESTED** [review]
Please accept suggestion to remove extra space

**Review by IEvangelist — APPROVED** [review]
Looks good to me, :shipit: when you're ready.

File: includes/core-changes/aspnetcore/5.0/middleware-database-error-page-obsolete.md [hunk 2]
```diff
@@ -57,6 +57,8 @@ Complete the following steps:
     }
     ```
 
+1. Add the `Microsoft.AspNetCore.Diagnostics.EntityFrameworkCore` NuGet package to the project file.
```
Comment (scottaddie) [inline includes/core-changes/aspnetcore/5.0/middleware-database-error-page-obsolete.md:60]:
```suggestion
1. Add the [Microsoft.AspNetCore.Diagnostics.EntityFrameworkCore](https://www.nuget.org/packages/Microsoft.AspNetCore.Diagnostics.EntityFrameworkCore) NuGet package to the project file.
```

Comment (scottaddie) [reply includes/core-changes/aspnetcore/5.0/middleware-database-error-page-obsolete.md:60]: This step should appear before the previous step. The `AddDatabaseDeveloperPageExceptionFilter` method lives in this NuGet package.

**Review by scottaddie — CHANGES_REQUESTED** [review]

**Review by scottaddie — APPROVED** [review]

**Pull request merged**